package.json

```diff
@@ -22,7 +22,8 @@
     "build": "webpack",
     "watch": "webpack --watch",
     "test": "npm-run-all --aggregate-output --parallel --continue-on-error test:* lint:*",
-    "test:mocha": "mocha --colors",
+    "test:client": "mocha --colors --require babel-polyfill --require babel-register test/client",
+    "test:server": "mocha --colors test/src",
```

Comment:
also bad rebase

Reply:
This is in the commit I'm removing anyway